### ISSUE TITLE
operating agreement

### ISSUE THREAD

**Issue by dertel** [dertel/old-issues#393]
**[Original report](https://bitbucket.org/davidartrecord/artrecord/issues/393) by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

find a template, start to tweak it before meeting michelle or other laywers.

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

@madeleineArtRec This has been a great read to help figure out whether we should use a template or work with a lawyer, I suggest reading it over:

https://businesssolutionsservices.com/2018/01/19/llc-operating-agreement/

It says that using templates and not hiring a lawyer at all can be fine, but it makes more sense for people with prior business experience.

*Even if a client bring him an existing operating agreement for review, it is often “more efficient for me [Dana] to start over, taking the client’s specific needs into account than to fix the existing OA.”*

Sounds like it's better to either 1) only use templates without a lawyer, or 2) work with a lawyer from the beginning

**Comment by dertel** [comment]
**Original comment by David Ertel (Bitbucket: [davidartrecord](https://bitbucket.org/davidartrecord), GitHub: [dertel](https://github.com/dertel)).**

----------------------------------------

@madeleineArtRec I've created a [Google Doc](https://docs.google.com/document/d/12tBse6nju8EEyH0aEDkoUZcK95nRgyisFKFry8DjmJw/edit#) that outlines some of the operating agreement terms and can also serve as a list of questions/issues to bring up with a lawyer. I based it on a legal advice article I found on google and added some other points.

Feel free to peruse and add your own questions or concerns, or reply to my questions. I think if we discuss these issues before our meeting with Michelle we would be more focused and get a lot more out of the meeting.
